### PR TITLE
Fset 1706 - Cumulative Evaluation results for latest phase results (showing failures from previous phases)

### PR DESCRIPTION
**Pull request by andrewrembrandt**

### CODE REVIEW

**Comment by benjstephenson** [comment]
LGTM, will run tests this evening and merge.
We need to make a note to implement the evaluation result roll-up at later stages too.

File: app/repositories/CumulativeEvaluationHelper.scala [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * limitations under the License.
+ */
+
+package repositories
+
+import model.persisted.SchemeEvaluationResult
+import reactivemongo.bson.{ BSONArray, BSONDocument, BSONString }
+
+trait CumulativeEvaluationHelper {
+  def cumulativeResultsForLatestPhase(latestResults: List[SchemeEvaluationResult]) = {
```
Comment (benjstephenson) [inline app/repositories/CumulativeEvaluationHelper.scala]: Can you rename this to make it clearer that it's returning the BSON document.

File: app/repositories/onlinetesting/OnlineTestEvaluationRepository.scala [hunk 2]
```diff
@@ -72,8 +74,12 @@ trait OnlineTestEvaluationRepository extends CommonBSONDocuments with ReactiveRe
       s"testGroups.$phase.evaluation" -> evaluation
     ).add(
       newProgressStatus.map(applicationStatusBSON).getOrElse(BSONDocument.empty)
+    ).add(
+      cumulativeResultsForLatestPhase(evaluation.result)
     ))
 
+    passMarkEvaluation.add()
```
Comment (benjstephenson) [inline app/repositories/onlinetesting/OnlineTestEvaluationRepository.scala]: What's this method call for?

**Comment by benjstephenson** [comment]
Some review comments and there are some failing admin acceptance tests but the number varies on each run.  Let's discuss in the morning; they seem to be assessor and event related.

File: app/repositories/CumulativeEvaluationHelper.scala (hunk 1 above)
Comment (andrewrembrandt) [inline app/repositories/CumulativeEvaluationHelper.scala]: Done

File: app/repositories/onlinetesting/OnlineTestEvaluationRepository.scala (hunk 2 above)
Comment (andrewrembrandt) [inline app/repositories/onlinetesting/OnlineTestEvaluationRepository.scala]: You were reviewing incomplete changes - only finished now!

**Comment by andrewrembrandt** [comment]
Afraid you were reviewing before I'd finished Ben (I left WIP on purpose) - just finished now

**Pull request merged**